bindings/icub.i

```diff
@@ -15,12 +15,15 @@
 #include <yarp/sig/all.h>
 using namespace yarp::os;
 using namespace yarp::sig;
-#include "cartesianController/ClientCartesianController.h"
-#include "gazeController/ClientGazeController.h"
+
+#include <iCub/iKin/iKinFwd.h>
+#include <iCub/iKin/iKinHlp.h>
+#include <iCub/iKin/iKinInv.h>
+#include <iCub/iKin/iKinIpOpt.h>
+#include <iCub/iKin/iKinSlv.h>
 %}
 
-%include "cartesianController/ClientCartesianController.h"
-%include "gazeController/ClientGazeController.h"
+%include <iCub/iKin/iKinFwd.h>
```

Comment:
I am a bit confused by this. You are including in the wrapper code all the headers of the iKin library, but then you are generating the bindings only for the classes contained in `iKinFwd.h` If you just want to wrap the classes in `iKinFwd.h`, then it should be sufficient to include `#include <iCub/iKin/iKinSlv.h>` in the earlier block. If instead you intend to wrap also the classes contained in the other heders, you should `%include` them as well.

Reply:
@towardthesea Was there any reason why you have chosen this way?

Reply:
@towardthesea any news on this?

Reply:
Hi, 
Sorry for late response. 
As I remember, there was no reason behind my choice. It is simply that I forgot to add other header files into the new binding file.
Thanks @traversaro for checking :+1:

Reply:
Done in https://github.com/robotology/icub-main/pull/479/commits/4d8ab44311f9a0ddfd7f99e69ea9c7891586e024 - credits to @towardthesea